cadc-permissions-client/build.gradle

```diff
@@ -14,17 +14,18 @@ sourceCompatibility = 1.8
 
 group = 'org.opencadc'
 
-version = '0.3.1'
+version = '0.3.2'
```

Comment:
Please revert this as it is unrelated to the rest of the PR.

Reply:
this whole file change still needs to be reverted

Reply:
That's done.  Did you see my earlier comment that it fixes the CI build though?  It's not related, true, but is it fixed elsewhere?